index.html

```diff
@@ -1,30 +1,5 @@
 <!DOCTYPE html>
-<!-- <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Tribute to Poet Allama Iqbal</title>
-    <link rel="stylesheet" href="styles.css">
-  </head>
-  <body>
-    <main id="main">
-      <h1 id="title">Allama Iqbal</h1>
-      <figure id="img-div">
-        <img id="image" src="https://img.dunyanews.tv/news/2017/November/11-09-17/news_big_images/413576_22615425.jpg" alt="Allama Iqbal">
-        <figcaption id="img-caption">Allama Iqbal</figcaption>
-      </figure>
-      <div id="tribute-info">
-           <p>Allama Iqbal was a poet, philosopher, and politician in British India who is widely regarded as having inspired the Pakistan Movement. He is considered one of the most important figures in Urdu literature, with literary work in both Urdu and Persian.</p>
-           <p>Iqbal is best known for his poetry, which emphasized the spiritual and religious values of Islam and the Muslim Ummah. His works include Asrar-e-Khudi, Rumuz-e-Bekhudi, and the Bang-e-Dra collection.</p>
-           <p>Born in Sialkot, Punjab, in 1877, Iqbal began his career as a lawyer, but soon turned to writing poetry and philosophical works. He was a strong advocate for the Muslim cause in India, and was instrumental in shaping the idea of a separate Muslim state, which ultimately led to the creation of Pakistan in 1947.</p>
-           <p>Throughout his life, Iqbal was deeply committed to promoting the values of Islam and the Muslim community. His poetry and philosophy continue to inspire people around the world, and he is widely considered one of the most important intellectual and cultural figures of the 20th century.</p>
-      </div>
-      <a id="tribute-link" href="https://en.wikipedia.org/wiki/Allama_Iqbal" target="_blank">Learn more about Allama Iqbal</a>
-    </main>
-  </body>
-</html> -->
-
```

Comment:
Please remove the commented parts of the code to ensure the implementation of good coding practice.